spec/models/blended_search_spec.rb

```diff
@@ -48,25 +48,28 @@
       end
 
       before do
-        suggestion = double('suggestion', text:'electrocoagulation')
+        suggestion = double('suggestion', text: 'electrocoagulation')
         expect(ElasticBlended).to receive(:search_for).
           with(hash_including(q: 'electro coagulation')).
           and_return(double(ElasticBlendedResults,
-                          results: [],
-                          suggestion: suggestion,
-                          total: 0))
+                            results: [],
+                            suggestion: suggestion,
+                            total: 0))
 
-        elastic_results = double(ElasticBlendedResults,
-                               results: [NewsItem.new],
-                               suggestion: double('suggestion', text:'electrocoagulation'),
-                               total: 1)
+        elastic_results = instance_double(ElasticBlendedResults,
+                                          results: [NewsItem.new(link: 'http://www.wh.gov/ns1',
```

Comment:
These new attributes for the `NewsItem` are the only changes in this file. The rest is Rubocop.